src/styles/_catalog.scss

```
@@ -1,12 +1,14 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  gap: 48px;
+  grid-template-columns: 487px;
+  grid-auto-columns: 200px;
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
     display: flex;
+    box-sizing: border-box;
```

```suggestion
  display: grid;
  gap: 48px;
  grid-template-columns: 487px;
  grid-auto-columns: 200px;
  max-width: 944px;
  margin: 0 auto;
  
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, minmax(200px, 1fr));
  }

  &__card {
    display: flex;
    box-sizing: border-box;
```
better to write media right here for better readability

---

pass all medias here